src/pages/my_page/user_profile/UserOrcid.tsx

```diff
@@ -126,17 +126,15 @@ export const UserOrcid = ({ user }: UserOrcidProps) => {
               rel="noopener noreferrer">
               {orcidUrl}
             </Typography>
+            <IconButton
+              color="error"
+              data-testid="button-confirm-delete-orcid"
+              onClick={toggleConfirmDialog}
+              aria-label={t('common.delete')}>
```

Comment:
Bra du tenker på aria-label! 
Men her er det ingen Tooltip, så skulle kanskje vært det i stedet/tillegg, eller evt bare sette `title`
```suggestion
              title={t('common.delete')}>
```

Reply:
Ser vi ikke hadde det før heller, men lurer på om det aller beste hadde vært å ha "Fjern ORCID-kobling" eller noe sånn her..